README.md

```diff
@@ -75,19 +75,54 @@ cd sample-pay-service
 
 ```bash
 npm install
-npm install -g ngrok
 ```
 
-3. Set your environment variables
+3. Install Twilio CLI
 
+The Twilio CLI allows you to manage your Twilio resources from your terminal or command prompt. You can use the Twilio CLI below to a provision phone number, configure the webhook, and run ngrok to expose your `localhost` to a public domain so the Twilio webhook can reach the expected endpoint. Alternatively, You can do this in the Twilio Console and by installing `ngrok` separately.
```

Comment:
I think we should either just link to the quickstart here or only show how to install the Twilio CLI using `npm`. Everything else seems confusing. Plus for this use case you'll have `npm` anyways installed

Reply:
👍